README.md

```diff
@@ -4,6 +4,8 @@
 
 - **Version 2.1 (breaking changes from 2.0.x)**
 
+  - 2.1.34: Refactor Den Haag textbox and textarea to Utrecht; add Single Select design tokens.
+  - 2.1.33: Added optional placeholder to all select elements.
```

Comment:
I think you didn't mean to duplicate this

Reply:
Nope 😄